Makefile

```diff
@@ -41,3 +41,4 @@ crosscompile:
 
 test:
 	@go test -cover ./...
+	@cd pkg/generators/models/testdata/cases/oneof_mapped_discriminator/expected && go test -cover ./...
```

Comment:
Why do we need this  `cd` and re-run the tests there?

Reply:
the tests is using a type that is generated and just checking if the can serialize/deserialize correctly. 

I couldn't get it to import that model and running `go test ./...` wasn't detecting it correctly. I think it might be because it is inside a folder with the `test` prefix, but i am not 100% sure. Jumping into the folder and running the tests from there seemed like the simplest approach.

Reply:
that is there so that we can test that the models are buildable https://github.com/contiamo/openapi-generator-go/pull/19